### PR TITLE
feat: adding support for Terraform Variables

### PR DESCRIPTION
**Pull request by away168**
Add support for Terraform Variables

Need help deciding what variable option to use - currently using `-u` since `-t` is taken by targets.

### CODE REVIEW

**Comment by away168** [comment]
i don't know how to write tests - so need help writing tests too.

**Comment by away168** [comment]
i  modified the variable tests to include type.

**Review by GiliFaroEnv0 — APPROVED** [review]
FIrst **Congratulation** on your first PR on this repo 🥳 🎉 

Good job! 😃 

left a minor comment on a file changed by mistake

File: node/src/main.js [hunk 1]
```diff
@@ -94,26 +95,27 @@ const getCommandOptions = (command, argv) => {
...
 };
 
-const parseEnvironmentVariables = (environmentVariables, sensitive) => {
+const parseVariables = (variables, sensitive, type) => {
   const result = [];
 
-  if (environmentVariables && environmentVariables.length > 0) {
-    logger.info(`Getting ${sensitive ? 'Sensitive ' : ''}Environment Variables from options:`, environmentVariables);
+  if (variables && variables.length > 0) {
+    logger.info(`Getting ${sensitive ? 'Sensitive ' : ''} Variables from options:`, variables);
```
Comment (RLRabinowitz) [inline node/src/main.js:102]: Maybe add a conditional `Environment Variables` vs `Terraform Variables` in this log?
Without this, the user would get the same `Getting Variables from options` log twice

File: node/src/config/commands.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const commands = {
     help: [
       {
         desc: 'Deploys an environment',
-        example: `$ env0 deploy -k <${API_KEY}> -s <${API_SECRET}> -o <${ORGANIZATION_ID}> -p <${PROJECT_ID}> -b <${BLUEPRINT_ID}> -e <${ENVIRONMENT_NAME}> -r [revision] -v [stage=dev]`
+        example: `$ env0 deploy -k <${API_KEY}> -s <${API_SECRET}> -o <${ORGANIZATION_ID}> -p <${PROJECT_ID}> -b <${BLUEPRINT_ID}> -e <${ENVIRONMENT_NAME}> -r [revision] -v [stage=dev] -u [tfvar=tfvalue]`
```
Comment (RLRabinowitz) [inline node/src/config/commands.js:12]: Maybe also add example for regular environment variables here

**Review by RLRabinowitz — APPROVED** [review]

File: node/src/commands/help.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ d88' \`88b \`888P"Y88b \`88.  .8'  888    888
 
 Self-Service Cloud Environments
 https://www.env0.com
+
```
[the remark this answers is not in the document]
Comment (away168) [reply node/src/commands/help.js]: fixed.

File: node/src/config/commands.js (hunk 2 above)
Comment (away168) [reply node/src/config/commands.js:12]: regular environment variables is still `-v [stage=dev]`

so not sure what you mean.

File: node/src/main.js (hunk 1 above)
Comment (away168) [reply node/src/main.js]: done.

**Comment by away168** [comment]
Addresses #78

**Pull request merged**